src/main/java/browserview/BrowserComponent.java

```diff
@@ -244,7 +246,11 @@ public void showFilterDocs() {
     public void showIssue(String repoId, int id, boolean isPullRequest, boolean isForceRefresh) {
         if (isPullRequest) {
             logger.info("Showing pull request #" + id);
-            runBrowserOperation(() -> driver.get(GitHubURL.getPathForPullRequest(repoId, id), isForceRefresh));
+            runBrowserOperation(() -> driver.get(
+                GitHubURL.getPathForPullRequest(repoId, id),
+                isForceRefresh,
+                (currentUrl, targetUrl) -> currentUrl.equalsIgnoreCase(targetUrl + "/commits") || currentUrl.equalsIgnoreCase(targetUrl + "/files")
+            ));
```

Comment:
I think rather than overloading `get`, it's better to do the check here. We can abstract it into a method, e.g. `isPrPageIgnoreTab(repoId, id)`, and if it's false, then we call the usual get.

I don't really like the idea of having a `get` with extra parameter to do extra check because as ChromeDriver middle man, it shouldn't be doing this.

Would it work with this?